app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryHistoryLocation.jsx

```diff
@@ -17,27 +17,21 @@ const ExpandedStoryHistoryLocation = ({ story, onClone, showHistory }) => (
     />
 
     <span className="input-group-btn">
-      <Clipboard
-        data-clipboard-text={storyUrl(story)}
-        component="button"
-        className="btn btn-default btn-clipboard"
-        button-title={I18n.t('story.events.copy_url')}
-      >
-        <img
-          src="/clippy.svg"
-          alt={I18n.t('story.events.copy_url')}
-          className="clipboard-icon"
-        />
-      </Clipboard>
+      <CopyToClipboard text={I18n.t('story.events.copy_url')}>
```

Comment:
Shouldn't this be?

```suggestion
      <CopyToClipboard text={storyUrl(story)}>
```

Reply:
Yes sure, you're right. Btw we need no translation because it is number and url. There is no necessity to translate number or urls.

Reply:
Done.